qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -27,13 +27,19 @@
 class IsingToQuadraticProgram:
     """Convert a qubit operator into a quadratic program"""
 
-    def __init__(self) -> None:
-        """Initialize the internal data structure."""
+    def __init__(self, linear: bool = False) -> None:
+        """Initialize the internal data structure.
+        Args:
+            linear: If linear is True, x^2 is treated as a linear term
+                since x^2 = x for x in {0,1}.
+                Else, x^2 is treat as a quadratic term
+        """
```

Comment:
```suggestion
        r"""
        Args:
            linear: If True, :math:`x^2` is treated as a linear term
                since :math:`x^2 = x` for :math:`x \in \{0,1\}`.
                Else, :math:`x^2` is treat as a quadratic term
        """
```

Reply:
@a-matsuo, see this for the `r` option and math

Reply:
Oh, I got it! Thank you very much for your kind and quick responses :D